tests/common/devices.py

```diff
@@ -268,6 +268,22 @@ def _get_platform_info(self):
                 result["hwsku"] = line.split(":")[1].strip()
             elif line.startswith("ASIC:"):
                 result["asic_type"] = line.split(":")[1].strip()
+
+        if result["platform"]:
+            platform_file_path = os.path.join('/usr/share/sonic/device', result["platform"], 'platform.json')
```

Comment:
There's a mix of single and double quotes throughout, can you make them all double quotes for consistency? (unless it's a character or string inside a string or something)